docker/gaffer-operation-runner/operation-runner/pom.xml

```diff
@@ -44,7 +44,7 @@
     <plugins>
       <plugin>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>3.2.2</version>
+        <version>3.3.3</version>
```

Comment:
Not sure why this changed to that version? `3.3.0` is the latest, there's no `3.3.3` and this is why CI fails.

Reply:
This was an accident due to find and replace 3.2.2 -> 3.3.3 for hadoop versions